src/Pages/DonatePage.jsx

```diff
@@ -0,0 +1,312 @@
+                data safe. The app's security features include data encryption,
+                purchase verification, multifactor authentication, and fraud
+                protection.
+              </Typography>
+            </AccordionDetails>
+          </Accordion>
+          <Accordion className={classes.faqAccordionStyle}>
+            <AccordionSummary
+              className={classes.faqSummary}
+              id="panel3a-header"
+              aria-controls="panel3a-content"
+              expandIcon={<ArrowDropDownOutlined className={classes.expand} />}
+            >
+              <Typography className={classes.faqSumText}>
+                I do not have a Venmo account, can I still donate online?
+              </Typography>
+            </AccordionSummary>
+            <AccordionDetails>
+              <Typography >
+                You will need a Venmo account to donate online. This can easily
+                be set up using only a mobile phone or email address. You can
+                then choose to either add funds to your Venmo account or link an
+                external payment method like a bank account or credit card.
+                <br />
+                <br />
+                You can sign up for a Venmo account here: https://get.venmo.com
+              </Typography>
+            </AccordionDetails>
+          </Accordion>
+          <Accordion className={classes.faqAccordionStyle}>
+            <AccordionSummary
+              className={classes.faqSummary}
+              id="panel4a-header"
+              aria-controls="panel4a-content"
+              expandIcon={<ArrowDropDownOutlined className={classes.expand} />}
+            >
+              <Typography className={classes.faqSumText}>
+                Can I make a donation by check?
+              </Typography>
+            </AccordionSummary>
+            <AccordionDetails>
+              <Typography >
+                Yes, please make the check out to Seamus Brugh and mail it to:
+                <br />
+                <br />
+                Seamus Brugh <br />
+                802 5th Ave N, unit 213 
+                <br />
+                Seattle, WA 98109
+              </Typography>
+            </AccordionDetails>
+          </Accordion>
+          <Accordion className={classes.faqAccordionStyle}>
+            <AccordionSummary
+              className={classes.faqSummary}
+              id="panel5a-header"
+              aria-controls="panel5a-content"
+              expandIcon={<ArrowDropDownOutlined className={classes.expand} />}
+            >
+              <Typography className={classes.faqSumText}>
+                Who should I contact if I have more questions?
+              </Typography>
+            </AccordionSummary>
+            <AccordionDetails>
+              <Typography>
+                You can send us an email at contact@cvp.com. We will get back to
```

Comment:
let's update this to `convictionvacation@gmail.com` please

Reply:
yes we can.